atomkraft/cli/__init__.py

```diff
@@ -16,11 +17,11 @@
 )
 
 GH_TEMPLATE = "gh:informalsystems/atomkraft"
+GH_REVISION = "rano/60-improve-code-quality"
```

Comment:
This needs to be fixed to `dev` before merging, probably?

Reply:
Yes. you're right :slightly_smiling_face: